examples/mnit_moles/src/moles.nit

```diff
@@ -227,51 +270,80 @@ class Screen
 
 		return false
 	end
+
+	# Draw the HUD as the topmost layer of the screen
+	fun draw_hud(display: Display)
+	do
+		var board = app.assets.points_board
+		board.scale = display_scale
+		display.blit(board, (32.0*display_scale).to_i, -10)
+
+		draw_score(display)
+	end
+
+	# Draw the score
+	fun draw_score(display: Display, score: nullable Int)
+	do
+		if score == null then score = game.points
+
+		# Draw the score itself
+		for img in app.numbers.imgs do img.scale = display_scale
+		display.blit_number(app.numbers, score,
+			(92.0*display_scale).to_i,
+			(172.0*display_scale).to_i)
+	end
 end
 
 redef class App
 
-	var screen: nullable Screen = null
+	# Main play screen
+	var screen = new Screen
+
+	# Image set generate by inkscape_tools
+	var assets = new DrawingImages
+
+	# Number to display score
```

Comment:
Is this that image where you put the `0..9` numbers used to print the score with a custom font?

If yes, the documentation can be improved.